stimulus/erlang.py

```diff
@@ -8,12 +8,13 @@ def intensity(aht, interval, rate):
 def erlang_b(server_count, intensity):
     ib = 1.0
     server_count = int(server_count) # need explicit cast here to prevent range TypeError
-    for i in range(0, server_count):
+    for i in xrange(0, server_count):
```

Comment:
I'm wondering the best way to do this since I'd like Stimulus to be PY3 compatible...

Reply:
`from past.builtins import xrange`